packages/blockchain/polygon/src/lib/services/polygon.kms.ts

```diff
@@ -19,12 +20,7 @@ export const polygonKmsService = (args: EvmBasedKMSServiceArgs) => {
         })
       }
 
-      if (
-        !transactionConfig.gasPrice ||
-        transactionConfig.gasPrice === '0' ||
-        transactionConfig.gasPrice === 0 ||
-        transactionConfig.gasPrice === '0x0'
-      ) {
+      if (!transactionConfig.gasPrice || BigNumber.from(transactionConfig.gasPrice).isZero()) {
```

Comment:
usually we are using bignumber.js
`import BigNumber from 'bignumber.js'`
check it please